src/build.py

```diff
@@ -343,6 +344,8 @@ def GitCloneFetchCheckout(self):
     """Clone a git repo if not already cloned, then fetch and checkout."""
     if os.path.isdir(self.src_dir):
       print '%s directory already exists' % self.name
+      proc.check_call(['git', 'remote', 'set-url', 'origin', self.git_repo],
```

Comment:
Actually, I'm not sure we want this. This will reset the `origin` remote even for local users. If we're going to be clever here, we should be more clever and if `origin` is the `external/llvm.org` mirror, then reset it, but otherwise don't touch it.